exercises/an_important_rule/problem.md

```diff
@@ -69,9 +69,8 @@ of functions that **all** print to the console.
    text `"OH NOES"`;
 2. Create a function `iterate` that prints the first argument
    (an integer) to it and then returns that argument + 1;
-3. Create a promise chain that wraps your iterate method using 
-  `Promise.resolve` then a series of iterations that attempts to perform 
-  `iterate` a total of 10 times.
+3. Create a promise chain that wraps your iterate method, then a series of 
+   iterations that attempts to perform `iterate` a total of 10 times.
```

Comment:
Hm – maybe it’s good to keep a mention of `Promise.resolve()`, but put the “using `Promise.resolve`” bit directly after `create a promise chain`?